src/client/src/App.jsx

```diff
@@ -1,28 +1,117 @@
-import "./App.css";
 import { Routes, Route, BrowserRouter as Router, } from "react-router-dom";
 import React, {useEffect, useState} from "react";
+import { FlashCardPage } from "./FlashCard/Page";
 
-const Placeholder = () => {
-    const [users, setUsers] = useState([]);
 
-    useEffect(() => {
-        fetch('/api/users').then( async (user) => {
-            const usersReturned = await user.json();
-            setUsers(Object.values(usersReturned)[0])
-        })
-    }, []);
+const DUMMY_FLASH_CARDS = [
+  {
+    id: 1,
+    term: 'andrew',
+    definition: 'spencer',
+  },
+];
 
-    return <div>{users.join(',')}</div>
-};
+// Make sure this is unique!
+let nextId = 2;
```

Comment:
This really would be handled by the server, but if needed, we could make this a function like `getNextId`